text/en/chapters/data-representation.md

```diff
@@ -682,6 +682,38 @@ The following interactive can be used to upload your own image, and experiment w
 
```

Comment:
So the new order would be:

- What's the big picture?
- Getting Started
- Numbers
- Text
- Images and Colours
- **Program Instructions**
- The whole story!
- Further reading